### PR TITLE
[WIP] Add "how to contribute" doc

### PR DESCRIPTION
**Pull request by hehouts**

### CODE REVIEW

**Comment by hehouts** [comment]
@taylorreiter please review

File: doc/13_contribute_on_github.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# How to edit the lessons in this webpage:
+
+Edit these lessons on [GitHub](https://github.com/dib-lab/dib_rotation)
+
+If you can't fix an error or make an addition right now, file an issue with the doc that need changing. Each doc (shows up as a tab or page on the website) is stored as a `.md` document within the `doc` folder.
```
Comment (taylorreiter) [inline doc/13_contribute_on_github.md:5]:
```suggestion
If you can't fix an error or make an addition right now, file an issue with the doc that needs changing. Each doc (shows up as a tab or page on the website) is stored as a `.md` document within the `doc` folder.
```

File: doc/13_contribute_on_github.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
++ `git clone https://github.com/dib-lab/dib_rotation.git` to clone the repo on to you computer
++ from inside that repo, make yourself a branch to work on: `git checkout -b my-awesome-branch`
++ make some changes in the markdown on your branch 
+    + `git add changed_file.md` for each file you change
+    + `git commit` to save changes
++ When you're satisfied with your changes, open a pull request to integrate these changes into the main branch:
+    + `git push --set-upstream origin my-awesome-branch`
+    + `git push` to upload updated files (all changes up to last commit)
+    + when you open GitHub again in a browser, it will prompt you to open a pull request
+        + if you are correcting a filed issue, its helpful to tag the issue number in the pull request (eg `fixes issue #1234`)
```
Comment (taylorreiter) [inline doc/13_contribute_on_github.md:17]:
```suggestion
        + if you are correcting a filed issue, it's helpful to tag the issue number in the pull request (e.g. `fixes issue #1234`)
```

File: doc/13_contribute_on_github.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    + this allows a tab to render for your doc in the side bar on the webpage
+    + place it in the list in the place you want it to appear on the webpage
+
+### View the updated rendered website locally:
++ make a conda env 
+    + `mamba create -n mkdocs mkdocs`  
+        + if you dont have mamba, use `conda create -n mkdocs mkdocs`
+    +  activate that environment: `conda activate mkdocs`
+    + `pip install mkdocs-material` to install the "material" theme
++ `cd` to the repo (`dib-rotation-project`, or wherever `mkdocs.yml` is sitting)
```
Comment (taylorreiter) [inline doc/13_contribute_on_github.md:34]:
```suggestion
+ `cd` to the repo (`dib_rotation`, or wherever `mkdocs.yml` is sitting) 
```

**Review by taylorreiter — APPROVED** [review]
few small suggestions, but otherwise LGTM! Thank you!

**Pull request merged**